ext/pdo/pdo_dbh.c

```diff
@@ -192,7 +192,7 @@ static char *dsn_from_uri(char *uri, char *buf, size_t buflen) /* {{{ */
 
 /* {{{ proto PDO::__construct(string dsn[, string username[, string passwd [, array options]]])
    */
-static PHP_METHOD(PDO, dbh_constructor)
+PHP_METHOD(PDO, dbh_constructor)
```

Comment:
Is this really needed? `PDO::__construct()` is its alias, but I don't see why we can't just define the constructor normally

Reply:
Don't see a reason for the `dbh_constructor` either, changing to `__construct` should be fine.